var/spack/repos/builtin/packages/openjdk/package.py

```diff
@@ -479,6 +487,36 @@ def install(self, spec, prefix):
         top_dir = "Contents/Home/" if platform.system() == "Darwin" else "."
         install_tree(top_dir, prefix)
 
+    @run_after("install")
+    @when("certs=system")
```

Comment:
I think you need to revert @alecbcs's change here because `when` and `run_after` don't get along. Sorry!  We can fix that separately.

Reply:
I reverted this and pushed it.